Dolt cause build error on some platform, so disable it.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [configuration]

Index: webkit-gtk-1.3.7+svnr72836-r72836-r0/configure.ac
===================================================================
--- webkit-gtk-1.3.7+svnr72836-r72836-r0.orig/configure.ac	2011-01-21 11:11:07.000000000 +0800
+++ webkit-gtk-1.3.7+svnr72836-r72836-r0/configure.ac	2011-01-21 11:11:15.000000000 +0800
@@ -82,8 +82,6 @@
 AC_DISABLE_STATIC
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
-# use dolt to speedup the build
-DOLT
 
 AC_PATH_PROG(FLEX, flex)
 if test -z "$FLEX"; then
